### PR TITLE
GST-457 Remove g_assert_*.

### PR DESCRIPTION
**Pull request by noneimb**

### CODE REVIEW

File: gst/parser/gstttmlparse.c [hunk 1]
```diff
@@ -835,7 +835,7 @@ subrip_fix_up_markup (gchar ** p_txt)
         break;
       default:
         GST_ERROR ("unexpected tag '%c' (%s)", *next_tag, next_tag);
-        g_assert_not_reached ();
```
Comment (khwaaj) [inline gst/parser/gstttmlparse.c:838]: `g_assert_not_reached()` is actually an [exception to the rule](https://docs.gtk.org/glib/func.assert_not_reached.html), i.e. they are disabled by `G_DISABLE_ASSERT `, so we can keep those.

I guess that makes this whole PR unnecessary?

**Review by khwaaj — COMMENTED** [review]
Just leaving a comment for now, without explicit approval or rejection, since the PR might not be relevant at all (see comment below).

File: gst/parser/gstttmlparse.c (hunk 1 above)
Comment (khwaaj) [reply gst/parser/gstttmlparse.c:838]: This is mostly a technicality, keeping `g_assert_not_reached()` likely yields a clearer log but all else is the same.

Comment (noneimb) [reply gst/parser/gstttmlparse.c:838]: Yes, I think it is unnecessary, thx.

**Comment by noneimb** [comment]
Closed as unnecessary.

**Pull request closed without merging**